autoware_perception_msgs/CMakeLists.txt

```
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.14)
+project(autoware_perception_msgs)
+
+find_package(ament_cmake_auto REQUIRED)
+ament_auto_find_build_dependencies()
+
+rosidl_generate_interfaces(${PROJECT_NAME}
+  "msg/TrafficLight.msg"
+  "msg/TrafficLightArray.msg"
+  "msg/TrafficLightElement.msg"
+  "msg/TrafficSignal.msg"
+  "msg/TrafficSignalArray.msg"
```

```suggestion
  "msg/TrafficLights.msg"
  "msg/TrafficLightElement.msg"
  "msg/TrafficSignal.msg"
  "msg/TrafficSignals.msg"
```
It's better to use plural case instead of adding `Array` suffix.

---

Many of the ros messages use `Array` sufix, so I think it's better to match them.
```
diagnostic_msgs/msg/DiagnosticArray
geometry_msgs/msg/PoseArray
common_interfaces/msg/MarkerArray
```

---

Sure, I've also created https://github.com/autowarefoundation/autoware-documentation/issues/296 to track it.